### ISSUE TITLE
Update Merriweather font to new version

### ISSUE THREAD

**Issue by ghost** [google/fonts#2943]
The font [Merriweather](https://fonts.google.com/specimen/Merriweather) 2 years ago chaged they repo to one, under they foundry at [Github/SorkinType](https://github.com/SorkinType/Merriweather). I would like to request an update to the latest version, or if I could do it, I more than happy to do it.

**Production notes:**
Don't forget to check if all open issues for Merriweather are fixed with this upgrade, and close them at the same tile.

**Comment by davelab6** [comment]
Will be fixed soon in #4077

**Comment by alxndr-w** [comment]
"soon"?

**Comment by chrissimpkins** [comment]
We are currently reviewing changes relative to the last production release in Fonts and will decide about the production push.  There are horizontal metrics changes and a move to lining proportional figures from oldstyle proportional figures.  Our team feels that both sets of changes are progression in a better direction for the family, but these changes will affect existing documents that use Merriweather. The H metrics impact will be an average line length that is *shorter* in the new version based on Marc Foley's diffenator2 review of the design changes.

**Comment by davelab6** [comment]
@RosaWagner wrote,

> Production notes:
> Don't forget to check if all open issues for Merriweather are fixed with this upgrade, and close them at the same tile.

Did anyone check this?

**Comment by davelab6** [comment]
Back on June 16th, @m4rc1e wrote:

> In 2018, Google Fonts commissioned Eben Sorkin to work on a variable font replacement for [Merriweather](https://fonts.google.com/specimen/Merriweather?query=merriweather). The work was [completed](https://github.com/google/fonts/pull/4077) in 2021 but it hasn’t been onboarded to Google Fonts yet because it's a huge project that requires a large investment in careful handling, and we only recently developed the tools needed to do so. I've now completed driving development of these tools and have started this work. This document reports my key findings. 
> 
> The horizontal metrics have changed significantly. I believe that the new variable fonts have better kerning, which is causing the new line length to be smaller.
> 
> ![kern change illustrated with toggling gif](https://github.com/google/fonts/assets/261579/1279d4de-880f-4b28-b4b1-0907d9d6d903)
> 
> In order to minimise the horizontal metrics differences, I considered changing the font’s optical size value to increase the line length. However, I found that the font’s optical size default value was already at the smallest allowable value in the font. 
> 
> Since we couldn’t increase the line length by making the opsz smaller, I presented the following workarounds in the pipeline meeting held on 2023-06-15:
> 
> 1. **Modify the kerning in the variable font,** to be more similar to the current one
> 2. **Release the family under a new name,** such as “Merriweather 2”
> 3. **Release as is,** since the kerning is an improvement
> 
> @davelab6 decided to go with option (3), so we’re hoping this document serves as evidence that we’re aware of this issue and the change is intentional.
> 
> Apart from the line length, the only other significant change is the figure style has been changed from old-style to lining
> 
> Current: Static Fonts with old-style default numerals:
> 
> <img width="484" alt="old-style default numerals" src="https://github.com/google/fonts/assets/261579/ec98ad4c-31d3-4b63-a635-ceb00c4e328d">
> 
> New: Variable Fonts, with lining default numerals:
> 
> <img width="353" alt="lining default numerals" src="https://github.com/google/fonts/assets/261579/5d2b9c53-9c80-4ae4-9499-e4b7d43a364b">
> 
> This change will bring the Merriweather family in line with our best practice recommendations and our other most-popular families: By default, we expect fonts to have lining proportional figures.

FYI @ebensorkin

**Comment by davelab6** [comment]
Since the old style numerals are available via OT Feature and CSS, I suggest we make sure to update the DESC file with info about how to use CSS to turn off kerning and turn on `osf` to replicate the v1 design

**Comment by kenmcd** [comment]
> 



> @RosaWagner wrote,
> 
> > Production notes:
> > Don't forget to check if all open issues for Merriweather are fixed with this upgrade, and close them at the same tile.
> 
> Did anyone check this?

There are still a lot of interpolation issues in the v2.200 files.
Most of the many I posted here are still in there: https://github.com/SorkinType/Merriweather/issues/540
And there are those plus others I missed in the Font Bakery report here: https://github.com/google/fonts/pull/4077#issuecomment-1604321642

**Comment by m4rc1e** [comment]
Confirmed on the interpolation issues

<img width="567" alt="Screenshot 2023-06-28 at 09 43 46" src="https://github.com/google/fonts/assets/7525512/cd44a979-8254-4716-8739-0edfb5b00f57">

wght=900, wdth=106, opsz=7

Interestingly, interpolation issues seem to appear when the wdth is greater than 100.

cc @ebensorkin

**Comment by EbenSorkin** [comment]
I am happy to re-kern the file to make it less drastic. I also think the
kerning that happens now in kern on is better than when I did this.

I am also happy to find and fix the interpolation problems.

**Comment by chrissimpkins** [comment]
@EbenSorkin Do you think that this is something that you will complete during Q3?

**Comment by EbenSorkin** [comment]
When does Q3 end?

-e.

**Comment by chrissimpkins** [comment]
This Friday :)

**Comment by EbenSorkin** [comment]
Yes. I’m hoping to have Au school and Briem school out with Simon and then
focus on Merriweather, the how to design for SSA document and supporting
the other African projects. I put five updates into the issues cue. They
all got GF African Pri updates. They are Gelasio, Alatsi, Alata, Calistoga,
and Voltaire. My hope now is that Merriweather can be concluded in 1-2
weeks.

**Comment by chrissimpkins** [comment]
status?

**Comment by EbenSorkin** [comment]
Au school and Breim are out of my hands and Merriweather is progressing.

I am fixing Merriweather for small subcontractor errors and revised Glyph
standards since they worked on it now. I think it will be mid-late Dec
before I have completed all the fixes and re-kerned it.

**Comment by chrissimpkins** [comment]
I have this tentatively slated for onboarding this quarter.  Is the development complete @EbenSorkin?

**Comment by EbenSorkin** [comment]
There is still some work to do on Italics I am about to swap into that
after I review small caps. I had been hoping to get Cyrillic improvements
in too but that may have to come by Summer.

I am still hoping to be done by end of Jan.

-e.

**Comment by chrissimpkins** [comment]
Status?

**Comment by m4rc1e** [comment]
I think Eben is still working on it.

**Comment by chrissimpkins** [comment]
Will move this to Q2. Thanks for letting us know!

**Comment by chrissimpkins** [comment]
@EbenSorkin how are things coming along with the updates?

**Comment by EbenSorkin** [comment]
I think I am getting very very close to done.

I'd like to think it will by next Friday or maybe even sooner.

**Comment by chrissimpkins** [comment]
Still working through FB fails on these updates.

**Comment by chrissimpkins** [comment]
Updated to P2 in Q4 2024

**Comment by EbenSorkin** [comment]
I’m hoping to close relevant fontbakery fails tonight. All interpolation
fails are resolved as of today.

**Comment by EbenSorkin** [comment]
Emma found 6 or so more issues so I am going to fix them this evening and
then hopefully that's it.

-e.

**Comment by emmamarichal** [comment]
@EbenSorkin Thank you for the corrections!
I think we can go ahead, I think @m4rc1e will do/update the PR next week, and I'll do a last revision there!

**Comment by emmamarichal** [comment]
solved in https://github.com/google/fonts/pull/7886
